docs/styles.md

```diff
@@ -13,6 +13,7 @@
     - [Equal Column Wide Tables](#wide-equal-column-table)
```

Comment:
It looks like somehow, in the confusion of the various branches :sweat_smile: , we accidentally backed out the changes in [this PR](https://github.com/openstax/k12-apps-raise/pull/91/files) (e.g. if we were to merge this PR we'd revert the changes there). The specific commits that seem to impact things:

* [b9eff6aa](https://github.com/openstax/k12-apps-raise/pull/89/commits/b9eff6aa0aecfe199163ac66a457542a14552378#diff-241f5064b93adeb15ef8ad6eb6db4f042d686554aef78819feea29a98a17db5d): This reverted the changes in `content.scss`
* [deb150b96](https://github.com/openstax/k12-apps-raise/pull/89/commits/deb150b96d5fe9729888f72ff450344cbb636583): This reverted the corresponding doc changes.

Since the commits :point_up: include other changes, just reverting them on this branch is likely a pain. One way to fix it could be to just cherry pick the commit from PR 91 to this branch, though that would impact history a bit:

```
$ git cherry-pick f3e8cc0e7369a32b36c9cff2b8e30982d697603b
```

In any case, something we'll need to address somehow prior to merging.